pages/resources/[page].js

```diff
@@ -1,89 +1,297 @@
 import { useEffect, useState } from 'react';
 import { useRouter } from 'next/router';
+import dynamic from 'next/dynamic';
+import { array, shape, string } from 'prop-types';
 import get from 'lodash/get';
-import isNumber from 'lodash/isNumber';
+import isFinite from 'lodash/isFinite';
+import omit from 'lodash/omit';
 import Content from 'components/Content/Content';
 import Head from 'components/head';
 import HeroBanner from 'components/HeroBanner/HeroBanner';
 import Pagination from 'components/Pagination/Pagination';
-import ResourceCard from 'components/Cards/ResourceCard/ResourceCard';
-import { getResourcesPromise } from 'common/constants/api';
+import ResourceSkeletonCard from 'components/Cards/ResourceCard/ResourceSkeletonCard';
+import {
+  getResourcesPromise,
+  getResourcesByCategories,
+  getResourcesByLanguages,
+  getResourcesBySearch,
+} from 'common/constants/api';
+import { Field, Formik } from 'formik';
+import Form from 'components/Form/Form';
+import Input from 'components/Form/Input/Input';
+import Button from 'components/Button/Button';
+import Select from 'components/Form/Select/Select';
+import Alert from 'components/Alert/Alert';
+import {
+  RESOURCE_SEARCH,
+  RESOURCE_SEARCH_BUTTON,
+  RESOURCE_RESET_BUTTON,
+} from 'common/constants/testIDs';
 import styles from '../styles/resources.module.css';
 
 const pageTitle = 'Resources';
 
-function Resources() {
+const ResourceCard = dynamic(() => import('../../components/Cards/ResourceCard/ResourceCard'), {
+  loading: () => <ResourceSkeletonCard numberOfSkeletons={1} />,
+});
```

Comment:
I changed it back to the way it was with no dynamic import. I also made it so the skeleton card will flash as long as the inputs are disabled (500ms) because of being submitted- this seems to alleviate the problem aaron noted about not noticing that the resources updated in the UI